src/OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -114,6 +117,8 @@ constexpr const char* kLogTimeFormat = "%Y-%m-%dT%H:%M:%E6S";
   }
 
 extern std::ofstream log_file;
+std::optional<int> GetLogFileLifetimeMinutes(std::string log_file_name);
```

Comment:
It looks like now we have knowledge about logfile format distributed between OrbitCore and OribtBase. I suggest introducing GetLogFileName function to this file and calling it from OrbitCore.

Alternatively you could move both of these functions to OrbitCore.

Reply:
Moved bot `CreateOrGetLogDir` and `GetLogPath` to OrbitBase. Thanks for the suggestion!